Sources/Core/AWSClientRuntime/Protocols/AWSJSON/AWSJSONError.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import protocol ClientRuntime.BaseError
+import enum ClientRuntime.BaseErrorDecodeError
+import class ClientRuntime.HttpResponse
+import class SmithyJSON.Reader
+
+public struct AWSJSONError: BaseError {
```

Comment:
The "base error" for use with AWS JSON 1.0 / 1.1.  Provides values for the common error fields.